public/components/register_model/register_model.tsx

```diff
@@ -290,17 +314,33 @@ export const RegisterModelForm = ({ defaultValues = DEFAULT_VALUES }: RegisterMo
       </EuiText>
     </>
   );
-
-  if (preTrainedModelLoading) {
-    return (
-      <EuiPanel>
-        {formHeader}
-        <EuiSpacer size="s" />
-        <EuiLoadingSpinner aria-label="Model Form Loading" size="l" />
-      </EuiPanel>
-    );
-  }
-
+  const [checked, setChecked] = useState(false);
+  const onChange = (e: any) => {
+    setChecked(e.target.checked);
+  };
+  const formFooter = (
```

Comment:
I think these two checkboxes can be separate to two form partials. We can add them to the form partials array.